backend/typescript/services/implementations/reviewService.ts

```diff
@@ -399,6 +480,266 @@ class ReviewService implements IReviewService {
 
     return result;
   }
+
+  async updateReviews(id: string, entity: ReviewRequestDTO): Promise<void> {
+    try {
```

Comment:
Also couldn't update the created_by field either

Reply:
Can you also check that the cover image is updated? I tried replacing the default text for the coverImage with another text and the field did not update in the books table row

Reply:
also the translator field. I couldn't add another translator to the review

Reply:
fixed